README.md

```diff
@@ -305,6 +305,70 @@ Run the tool through the context menu on a file or directory:
 
 </details>
 
+### Workflow for GitHub Actions
+
+<details>
+
+#### What does it do?
+
+This workflow will convert every JPG, JPEG and PNG file into AVIF and WebP.
```

Comment:
As far as I know JPG and JPEG are the same thing:

https://en.wikipedia.org/wiki/JPEG

As I see originally you meant file extensions here, and it was fine. But now it's written as file formats, and I think we should leave JPEG only.

There're several places where JPG & JPEG names are used in this file. Let's fix them all, shall we?

Reply:
True, kinda stupid of me to use both jpg and jpeg names :P